Bottom: 31c0cf47f562afc290796ba3e9791f672a7d5d8b
Top:    95b121ec32dc0d0d10c0de528bba1ae1ed0075b1
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-07-19 14:21:17 -0700

Refresh of cma-rm-pd

---

diff --git a/src/cma.c b/src/cma.c
index e2772fa..65b52bf 100755
--- a/src/cma.c
+++ b/src/cma.c
@@ -378,8 +378,8 @@ out:
 static void ucma_put_device(struct cma_device *cma_dev)
 {
 	pthread_mutex_lock(&mut);
-	if (!--cma_dev->verbs_refcnt)
-		ibv_dealloc_pd(cma_dev_array[i].pd)
+	if (!--cma_dev->refcnt)
+		ibv_dealloc_pd(cma_dev->pd);
 	pthread_mutex_unlock(&mut);
 }
